tests/pub-external-links/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,9 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>External Links should be opened a new browser instance</title>
+  </head>
+  <body>
+    <p>Test passes if following link is opened in a new browser instance.</p>
```

Comment:
How does a tester *tell* that the external link opens in a new browser instances? Like, what are the characteristics?

Reply:
yeah, good question. I would image either a new browser tab or a new browser window.